### PR TITLE
Quick Fix for the Sample Availability

### PR DESCRIPTION
**Pull request by junczhu**
image

By doing quick test I found in line 83 the value of _x.From.Id_ will be either _DirectLineSampleClientUser_ or _directline=csharp_

And in this case we need set value to _directline-csharp_ of an expected output.

BR,
CharingJC

### CODE REVIEW

**Review by EricDahlvang — CHANGES_REQUESTED** [review]
Hello @CharingJC 

The botId should be coming from app.config:
https://github.com/microsoft/BotFramework-DirectLine-DotNet/blob/main/samples/core-DirectLineWebSockets/DirectLineClient/App.config#L5  

Maybe the Readme.md should state this.  Would you care to update these two readmes, instructing users to add their bot's id to the app.config?

**Comment by CharingJC** [comment]
> Hello @CharingJC
> 
> The botId should be coming from app.config:
> https://github.com/microsoft/BotFramework-DirectLine-DotNet/blob/main/samples/core-DirectLineWebSockets/DirectLineClient/App.config#L5
> 
> Maybe the Readme.md should state this. Would you care to update these two readmes, instructing users to add their bot's id to the app.config?

Sure

File: samples/core-DirectLineWebSockets/README.md [hunk 1]
```diff
@@ -82,7 +83,7 @@ private static void WebSocketClient_OnMessage(object sender, MessageEventArgs e)
 {
     var activitySet = JsonConvert.DeserializeObject<ActivitySet>(e.Data);
     var activities = from x in activitySet.Activities
-        where x.From.Id == botId
+        where x.From.Id == "directline-csharp"
```
Comment (EricDahlvang) [inline samples/core-DirectLineWebSockets/README.md:86]: This line should not be changed, since it is pulled from app.config

Comment (CharingJC) [reply samples/core-DirectLineWebSockets/README.md:86]: Hello Eric, 

Thank you for the sharing.
However base on my test the value of x.From.Id is either  directline-csharp or DirectLineSampelClientUser
Please check the highlight part of the Image.
image

BR,
Juncheng

Comment (EricDahlvang) [reply samples/core-DirectLineWebSockets/README.md]: Thank you for adding the notes to the readme.

Regarding botid, have you tried this with a different bot?  one with a bothandle other than "directline-csharp"?

Comment (CharingJC) [reply samples/core-DirectLineWebSockets/README.md]: Thank you for the response Eric.
Here is the test result with a bot named 'one-more-directline', it seems that the From.Id property is replaced by the bothandle name.

image

**Comment by junczhu** [comment]
closed for long running PR

**Pull request closed without merging**